vmdk_plugin/utils/test_util/VolumeCreationTestUtil.go

```diff
@@ -49,3 +50,121 @@ func DeleteVolume(name string, ip string) ([]byte, error) {
 func InvokeCommand(remoteHostIP string, cmd string) ([]byte, error) {
 	return exec.Command("/usr/bin/ssh", append(SSH_OPTS, "root@"+remoteHostIP, cmd)...).CombinedOutput()
 }
+
+// returns disk format of volume using Admin cli
+func GetDiskFormat_AdminCli(volName string, hostname string) string {
+	cmd := "/usr/lib/vmware/vmdkops/bin/vmdkops_admin.py ls -c volume,disk-format | grep " + volName
+	io := ExecuteCmd(cmd, hostname)
+	if strings.Contains(io, "stty: standard input: Inappropriate ioctl for device") {
```

Comment:
#58 - #60 can be replaced by appending ```2>/dev/null``` to the command as it is being ignored here